test/functional/environments_controller_test.rb

```diff
@@ -1,16 +1,12 @@
 require 'test_helper'
+require 'functional/shared/basic_rest_response_test'
 
 class EnvironmentsControllerTest < ActionController::TestCase
-  test "should get index" do
-    get :index, {}, set_session_user
-    assert_response :success
-    assert_not_nil assigns(:environments)
```

Comment:
Fixed, now it's even better as the shared module tests this in this class & others